packages/metrics/src/config/routes.ts

```diff
@@ -293,6 +295,22 @@ export const getRoutes = () => {
         tags: ['api']
       }
     },
+    {
+      method: 'POST',
+      path: '/events/downloaded',
+      handler: declarationDownloadedHandler,
+      config: {
+        tags: ['api']
+      }
+    },
+    {
+      method: 'POST',
+      path: '/events/viewed',
+      handler: declarationViewedHandler,
+      config: {
+        tags: ['api']
+      }
+    },
```

Comment:
@KaliGreen We will need routes for the birth/death archive/reinstate events as well

Reply:
added